Adding a what's new page for MetaMask docs (#1179)
MetaMask/metamask-docs: src/pages/whats-new.md

---

> title: "What's new in the MetaMask documentation"

Reviewer: ```suggestion
title: "What's new in the documentation"
```

description: Summary of the latest updates to this documentation.
---

# What's new in the documentation

> The latest MetaMask documentation updates are listed by the month they were implemented. For a comprehenesive list of recent product changes, visit the [MetaMask developer page](https://metamask.io/developer/) or see the [release notes](https://github.com/MetaMask/metamask-sdk/releases).

Reviewer: ```suggestion
The latest major MetaMask documentation updates are listed by the month they were added to the documentation. For a comprehensive list of recent product changes, visit the **Release Notes** section at the bottom of the [MetaMask developer page](https://metamask.io/developer/).
```

> ## February 2024
> - Added information about [ERC-4337](https://metamask.io/news/latest/account-abstraction-past-present-future/) - a unified standard that makes [account abstraction easier](/snaps/features/custom-evm-accounts/security). [(#1167)](https://github.com/MetaMask/metamask-docs/pull/1167)
> - Documented using [Snaps environment variables](/snaps/how-to/use-environment-variables). [(#1149)](https://github.com/MetaMask/metamask-docs/pull/1149)
> - Documented [Snaps static file API](/snaps/features/static-files). [(#1138)](https://github.com/MetaMask/metamask-docs/pull/1138)
> - Documented Snaps [unencrypted state](/snaps/reference/snaps-api/#snap_managestate). [(#1124)](https://github.com/MetaMask/metamask-docs/pull/1124)
> - Restructured [Snaps documentation](/snaps) for improved readability. [(#1129)](https://github.com/MetaMask/metamask-docs/pull/1129)
> - Added new [Snaps architecture diagrams](/snaps/learn/about-snaps). [(#1123)](https://github.com/MetaMask/metamask-docs/pull/1123)

Reviewer: ```suggestion
## February 2024

- Documented [account abstraction via ERC-4337](/snaps/features/custom-evm-accounts/#account-abstraction-erc-4337). ([#1167](https://github.com/MetaMask/metamask-docs/pull/1167))

- Documented [how to use Snaps environment variables](/snaps/how-to/use-environment-variables). ([#1149](https://github.com/MetaMask/metamask-docs/pull/1149))

- Documented [Snaps static files](/snaps/features/static-files). ([#1138](https://github.com/MetaMask/metamask-docs/pull/1138))

- Restructured [Snaps documentation](/snaps) for improved readability. ([#1129](https://github.com/MetaMask/metamask-docs/pull/1129))

- Added [Snaps architecture diagrams](/snaps/learn/about-snaps). ([#1123](https://github.com/MetaMask/metamask-docs/pull/1123))

- Documented [`snap_getClientStatus`](/snaps/reference/snaps-api/#snap_getclientstatus). ([#1119](https://github.com/MetaMask/metamask-docs/pull/1119))

- Documented [how to detect multiple wallets via EIP-6963](/wallet/how-to/detect-wallet/multiple-wallets). ([#1094](https://github.com/MetaMask/metamask-docs/pull/1094))
```

> ## January 2024

Reviewer: Please be attentive to your spacing for source code readability. Always add line breaks around headers and don't use multiple line breaks anywhere.

Also, remember the [column limit](https://docs-template.consensys.io/contribute/format-markdown#column-limit).

> 
> ## January 2024
> - Documented [`snap_getClientStatus`](/snaps/reference/snaps-api/#snap_getclientstatus). [(#1119)](https://github.com/MetaMask/metamask-docs/pull/1119)
> - Updated [wallet interoperability](/wallet/concepts/wallet-interoperabilty) and [multiple wallets](/wallet/how-to/detect-wallet/multiple-wallets) documentation. [(#1094)](https://github.com/MetaMask/metamask-docs/pull/1094)
> - Added a [Snap localization tutorial](/snaps/features/localization). [(#1093)](https://github.com/MetaMask/metamask-docs/pull/1093)
> - Improved [Snaps concepts](/snaps/learn/about-snaps) documentation and [API reference](/snaps/reference/snaps-api). [(#1098)](https://github.com/MetaMask/metamask-docs/pull/1098)

Reviewer: ```suggestion
```

The first two items were merged in February, not January. Which leaves us with very few updates in January, so we can just leave January off and start in February.
